### PR TITLE
Feature/issue 29 edit response not found search user/id

### LINKED ISSUE #29: Edit response not found search user/id
**Priority :**
1

**Duration :** 
1

**Description:**
Edit response not found search user/id

**File added / modified:**

**Additional context**

Add any other context or screenshots here.

### PR DESCRIPTION
**Pull request by vanmarcke**
Issue [#29]

Edit response not found search user/id

### CODE REVIEW

File: src/Manager/ProductsManager.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ public function getProductsList(): array
     /**
      * {@inheritdoc}
      */
-    public function getProductId($id): Products
+    public function getProductId($id)
```
Comment (antoinetrouve) [inline src/Manager/ProductsManager.php]: il faut garder le typage

File: src/Manager/ProductsManagerInterface.php [hunk 2]
```diff
@@ -18,5 +18,5 @@ public function getProductsList(): array;
      *
      * @param $id Main product reference
      */
-    public function getProductId($id): Products;
+    public function getProductId($id);
```
Comment (antoinetrouve) [inline src/Manager/ProductsManagerInterface.php]: idem

**Review by antoinetrouve — APPROVED** [review]

File: src/Manager/ProductsManager.php (hunk 1 above)
Comment (vanmarcke) [reply src/Manager/ProductsManager.php]: done

File: src/Manager/ProductsManagerInterface.php (hunk 2 above)
Comment (vanmarcke) [reply src/Manager/ProductsManagerInterface.php]: done

**Pull request merged**